src/components/Tool/FixSendNewsletter/FixSendNewsletter.test.tsx

```diff
@@ -159,15 +155,30 @@ describe('FixSendNewsletter', () => {
 
       const newsletterDropdown = getAllByRole('combobox')[0];
       await waitFor(() => {
-        expect(newsletterDropdown).toHaveDisplayValue([initialNewsletterValue]);
+        expect(
+          within(newsletterDropdown).getByText(initialNewsletterValue),
+        ).toBeInTheDocument();
+        userEvent.click(newsletterDropdown);
+        userEvent.click(getByText(newNewsletterValue));
       });
-      userEvent.selectOptions(newsletterDropdown, newNewsletterValue);
       userEvent.click(getAllByRole('button', { name: 'Confirm' })[0]);
       await waitFor(() => {
         expect(mockEnqueue).toHaveBeenCalledWith('Newsletter updated!', {
           variant: 'success',
         });
-        expect(queryByText(name)).not.toBeInTheDocument();
+      });
+      await waitFor(() => {
+        expect(mutationSpy.mock.calls[1][0]).toMatchObject({
+          operation: {
+            operationName: 'UpdateContactNewsletter',
+            variables: {
+              accountListId: 'account-id',
+              attributes: {
+                sendNewsletter: 'PHYSICAL',
+              },
+            },
+          },
+        });
```

Comment:
You should check that the contact gets removed from the page after a successful request.

Reply:
Since I'm using `refetchQueries` it is hard to test that an item was removed. Because refetchQueries brings in the same mock data. Do you have any ideas on how to test this?

Reply:
If the test is too hard, move onto the next item. We can come back.